rocky/tools/forms/base.py

```diff
@@ -63,11 +63,18 @@ class ObservedAtForm(BaseRockyForm):
     observed_at = forms.DateField(
         label=_("Date"),
         widget=DateInput(format="%Y-%m-%d"),
-        initial=lambda: datetime.datetime.now(tz=datetime.timezone.utc),
+        initial=lambda: datetime.now(tz=timezone.utc),
```

Comment:
This was already in the existing code, but given that it is DateField I think the initial value should also be a date and not a datetime.